CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
   - Update Gradle to `5.6.2` version
   - Update Kotlin to `1.3.50` version
-  - Set default ktlint version to `0.34.2`
+  - Set default ktlint version to `0.35.0`
```

Comment:
This is wrong place to update changelog. It should be in "Unreleased" section, subsection "changed".

Reply:
Ok, I'll fix it

Reply:
@Tapchicoma, done